geometry_msgs/Point[] Position_XYZ
uint16 center_pixel_x
uint16 center_pixel_y
float64 max_x
float64 max_y
uint8 counter